README.md

```diff
@@ -27,6 +33,17 @@ To check if everything worked fine, you can do:
 
 and it should not give any error messages.
 
+To run the tests using `pytest`, you need to have the cloned mne-python repository
```

Comment:
I would move it to the contributions section. It's not relevant for regular users.

Reply:
+1